src/MainPage/ChannelsList.module.scss

```diff
@@ -1,111 +1,103 @@
-            padding-left: 2px;  
-        }
-        
-        .channelsVisible {
-            transform: rotate(90deg);
-            transition: 0.3s;
-        }
-        .plusIcon {
-            font-size: 22px;
-            margin-left: auto;
-            pointer-events: none;
-            opacity: 0;
-            transition: opacity 0.3s;     
-            
-        }
-        .channelsNotVissible {   
-            transition: 0.3s;
-        }
-        &:hover{
-            color: white;
-            .plusIcon {
-                pointer-events: all;
-                opacity: 1;     
-            }
-        }
-      
+    .channelsVisible {
+      transform: rotate(90deg);
+      transition: 0.3s;
+    }
+    .plusIcon {
+      font-size: 22px;
+      margin-left: auto;
+      pointer-events: none;
+      opacity: 0;
+      transition: opacity 0.3s;
+    }
+    .channelsNotVissible {
+      transition: 0.3s;
     }
-    .channelContainer {
-        width: 100%;
-        display: flex;
-        align-items: center;
-        justify-content: flex-start;
-        box-sizing: border-box;
-        padding-right: 20px;
-        // overflow : hidden;
-        // white-space: nowrap;
-        // text-overflow: ellipsis;
-    
-        .channelsListTitle {
-        width: 100%;
-        // position: relative;
-        box-sizing: border-box;
-        overflow : hidden;
-        white-space: nowrap;
-        text-overflow: ellipsis;
-        font-weight:400;
-        padding-left: 14%;
-        margin: 2px 0;
-        cursor: pointer;
-        &:hover {
-            color: white;
-        }
+    &:hover {
+      color: white;
+      .plusIcon {
+        pointer-events: all;
+        opacity: 1;
+      }
+    }
+  }
+  .channelContainer {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    box-sizing: border-box;
+    padding-right: 20px;
 
-        }
-        button {    
-            padding: 0;
-            margin: 2px;
-            border: none;
-            text-decoration: none;
-        }
-        .deleteIconContainer {
-            position: relative;
-            // top: 4px;
-            color:rgb(188,171,188);
-            font-size: 22px;
-            .deleteIcon {
-                position: relative;
-                top: 4px;
-            }
-            &:hover {
-            color: white;
-            }
-        }
+    .channelsListTitle {
+      width: 100%;
+      box-sizing: border-box;
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      font-weight: 400;
+      padding-left: 14%;
+      margin: 2px 0;
+      cursor: pointer;
+      &:hover {
+        color: white;
+      }
     }
-    .currentChannel {
-        background-color: rgb(33, 80, 196);
+    button {
+      padding: 0;
+      margin: 2px;
+      border: none;
+      text-decoration: none;
+    }
+    .deleteIconContainer {
+      position: relative;
+      color: rgb(188, 171, 188);
```

Comment:
Wyciągnij kolory do zmiennych, których potem tu użyjesz,
możesz to zrobić dla wszystkich kolorów, które zaczynają się od rgb.
Które nie są zapisane jednym słowem jak np white itp.